src/vs/workbench/contrib/editSessions/common/editSessions.ts

```diff
@@ -82,9 +82,9 @@ export const EDIT_SESSIONS_SIGNED_IN = new RawContextKey<boolean>(EDIT_SESSIONS_
 
 export const EDIT_SESSIONS_CONTAINER_ID = 'workbench.view.editSessions';
 export const EDIT_SESSIONS_DATA_VIEW_ID = 'workbench.views.editSessions.data';
-export const EDIT_SESSIONS_TITLE = localize('edit sessions', 'Edit Sessions');
+export const EDIT_SESSIONS_TITLE = localize('cloud changes', 'Cloud Changes');
 
-export const EDIT_SESSIONS_VIEW_ICON = registerIcon('edit-sessions-view-icon', Codicon.cloudDownload, localize('editSessionViewIcon', 'View icon of the edit sessions view.'));
+export const EDIT_SESSIONS_VIEW_ICON = registerIcon('edit-sessions-view-icon', Codicon.cloudDownload, localize('editSessionViewIcon', 'View icon of the roaming changes view.'));
```

Comment:
What is the different between `roaming changes`, `cloud changes`, and `working changes`?

Reply:
There shouldn't be `Roaming Changes`, thanks for catching those--I got feedback at the UX sync that `Cloud Changes` was clearer than `Roaming Changes`. `Working changes` refers to the changes before they are stored in the cloud (it wouldn't be accurate to call them cloud changes at that point). The reason for this change is to avoid using the term edit sessions in the product surface because the name is an implementation detail and ambiguous/not intuitive to users.